Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -49,7 +51,7 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor,
    * The access token to use to access the Cesium ion resource.
    */
   UPROPERTY(
-      EditAnywhere,
+      EditAnywhere, BlueprintReadOnly,
```

Comment:
Why does this need to be `BlueprintReadOnly`? I saw your comment "Added BueprintReadOnly flag to CesiumIonToken to be able to check that tokens have been set..." but I'm not quite sure what you mean.

Reply:
I just wanted to make sure the property was accessible from a blueprint graph, but in readonly mode... 
In my code, I do a sanity check by iterating on all Cesium3DTilesets to check if the token is not Empty. If I encounter an empty one, I display a warning message in the app.

Reply:
Oh ok, so if we make it read/write that would be just as good for your purposes, right?

Reply:
Sorry if I misunderstood something here due to a lack of context, but I just stumbled over the statement *"iterating on all Cesium3DTilesets to check if the token is not Empty"* - On the one hand, I could simply assume that this is only intended for tilesets that are *not* loaded from local files, but on the other hand, this raises the question of why the distinction between "ion and local file" should be made ... (?)

Reply:
@javagl, this is of course intended only for ion assets because the token is only available for this kind of assets.  It helps to manage access of remote applications. When using local files, there is no mechanism for access rights. 
My request was just to be able to do some sanity checks to make sure the developer put the right tokens in its UE level. This is really application-specific, and the current Cesium API prevented from doing that, the value was not accessible from within a BP graph.

Reply:
Usually, there should not be a distinction between a "file-based asset" and an "ion-based asset" on that level. **If** we expose a certain functionality (like obtaining the token), then it will never be possible to remove that functionality in a later release. I'm always hesitant to ~"make something `public` just for convenience". There should be strong, technical reason, and one should be absolutely sure that this is the right thing to do, and will stay like that forever. But.... it's merged now, so ... no need to further dissect this.